### DISCUSSION TITLE
Send email explaining why RA could not be voted

### DISCUSSION THREAD

**Discussion by TomH-OpsDev** [AdhocAdam/smletsexchangeconnector#369] [Ideas]
Send an explanatory email as to why an attempt to [approve] or [reject] an RA was not successful.  Or perhaps merely that it was unsuccessful.

https://github.com/AdhocAdam/smletsExchangeConnector/edit/v1.4b-TH/smletsExchangeConnector.ps1#L917

**Comment by AdhocAdam** [comment]
With the HTML template customization introduced on #69 and ultimately in v.1.4.5 (#78) what if we leveraged the custom actions portion of the connector? That way styling and formatting would be preserved throughout connector upgrades?
